webapp/js/lib/utils.js

```
@@ -0,0 +1,129 @@
+        return -1;
+    }
+}
+
+// Return human-readable filesize
+function getHumanReadableSize(size) {
+    if (_.isUndefined(size)) return;
+    return filesize(size, {base: 10});
+}
+
+var validAmount  = function(n) {
+    return !isNaN(parseFloat(n)) && isFinite(n);
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)